infra/modules/monitoring/main.tf

```diff
@@ -57,6 +57,7 @@ resource "aws_cloudwatch_metric_alarm" "high_app_response_time" {
   statistic           = "Average"
   threshold           = 0.2
   alarm_description   = "High target latency alert"
+  treat_missing_data  = "notBreaching"
```

Comment:
I think we'd want to use `ignore` instead.

For example, we have the following 5-minute intervals: `[good, error, error, missing, error, error, missing, error]` - Clearly, something is wrong, the missing value is just a sampling issue (the same one causing this noisy alert). `notBreaching` would be treated as `good` when we'd rather just see the errors.

Reply:
Sure I can try that. I will test it.

Reply:
Actually, a test would take quite some time. I feel fine merging this without an explicit test.